src/estimators/bloom_filters.py

```diff
@@ -435,6 +436,19 @@ def __call__(self, bloom_filter):
     return new_filter
 
 
+def get_probability_of_flip(epsilon, num_hashes):
+  """Get the flipping probability from the privacy epsilon.
+
+  Args:
+    epsilon: the differential privacy parameter.
+    num_hashes: the number of hash functions used by the bloom filter.
+
+  Returns:
+    The flipping probability.
+  """
+  return 1 / (1 + math.exp(epsilon / num_hashes))
```

Comment:
Maybe, add a comment to explain: The per bucket flipping probability should factor in the id sensitivity.  Multiple hashes assign an id to num_hashes bit buckets, and thus increase the sensitivity of an id to num_hashes.

Reply:
Ack.